package.json

```diff
@@ -15,8 +15,9 @@
     "ember-source-3-25": "npm:ember-source@3.25.4",
     "ember-source-3-28": "npm:ember-source@3.28.1",
     "fast-glob": "^3.2.7",
-    "glimmer-compiler-ember-source-3-28": "npm:@glimmer/compiler@0.80.0",
     "glimmer-compiler-ember-source-3-24": "npm:@glimmer/compiler@0.65.3",
+    "glimmer-compiler-ember-source-3-28": "npm:@glimmer/compiler@0.80.0",
+    "glimmer-compiler-experiment": "link:../../glimmerjs/glimmer-vm/dist/@glimmer/compiler",
```

Comment:
can you add this to the repo as a workspace?

Reply:
Ah, that's a good thought. Will do!